Formula/xapian.rb

```diff
@@ -39,6 +28,9 @@ class Xapian < Formula
   deprecated_option "ruby" => "with-ruby"
 
   depends_on :python => :optional
+  depends_on "sphinx-doc" if build.with? "python" => :build
```

Comment:
More specifically, is there no valid use case where you'd want documentation but not the bindings?